src/pe_source/intelx_identity.py

```diff
@@ -0,0 +1,289 @@
+            LOGGER.error(e)
+            return 1
+        return 0
+
+    def query_identity_api(self, domain, start_date, end_date):
+        """Create an initial search and return the search id."""
+        url = f"https://3.intelx.io/accounts/csv?selector={domain}&k={api_key}&datefrom={start_date}&dateto={end_date}"
+        payload = {}
+        headers = {}
+        attempts = 0
+        while attempts < 5:
+            try:
+                response = requests.request("GET", url, headers=headers, data=payload)
+                break
+            except requests.exceptions.Timeout:
+                time.sleep(5)
+                attempts += 1
+                if attempts == 5:
+                    LOGGER.error("IntelX identity is not responding. Exiting program.")
+                    sys.exit()
+                LOGGER.info("IntelX Identity API response timed out. Trying again.")
+            except Exception as e:
+                LOGGER.error(f"Error occured geting search id: {e}")
+                return 0
+        LOGGER.info("Acquired search id.")
+        time.sleep(5)
+        return response.json()
+
+    def get_search_results(self, id):
+        """Search IntelX for email leaks."""
+        url = f"https://3.intelx.io/live/search/result?id={id}&format=1&k={api_key}"
+
+        payload = {}
+        headers = {}
+        attempts = 0
+        try:
+            response = requests.request("GET", url, headers=headers, data=payload)
+        except requests.exceptions.Timeout:
+            time.sleep(5)
+            attempts += 1
+            if attempts == 5:
+                LOGGER.error("IntelX identity is not responding. Exiting program.")
+                sys.exit()
+            LOGGER.info("IntelX Identity API response timed out. Trying again.")
+        except Exception as e:
+            LOGGER.error(f"Error occured geting search results: {e}")
+            return 0
+        response = response.json()
+
+        return response
+
+    def find_credential_leaks(self, domain_list, start_date, end_date):
+        """Find leaks for a domain between two dates."""
+        all_results_list = []
+        for domain in domain_list:
+            LOGGER.info("Finding credentials leaked associated with " + domain)
+            response = self.query_identity_api(domain, start_date, end_date)
+            if not response:
+                continue
+            search_id = response["id"]
+            while True:
+                results = self.get_search_results(search_id)
+                if not results:
+                    break
+                if results["status"] == 0:
+                    current_results = results["records"]
+                    if current_results:
+                        # Add the root_domain to each result object
+                        LOGGER.info(
+                            f"Intelx returned {len(current_results)} more credentials for {domain}"
```

Comment:
Use consistent capitalization:
```suggestion
                            f"IntelX returned {len(current_results)} more credentials for {domain}"
```